ONSdigital/takeon-ui
# Spp2280/feature/round adjusted to 2dp

# Description
Javascript has been added to the BaseFormView.html template which will round adjusted values to 2 decimal places. This is a tactical fix, as Dave puts it a hack, so it will not extend well to surveys other than 023.
https://collaborate2.ons.gov.uk/jira/browse/SPP-2280
No refactoring.

# How to test?
Manual testing was done to ensure that the rounding occurred on all required views, regardless of the data displayed. Testing was done to ensure there wasn't unforeseen negative impacts on unchanged views or unfilled pages. 
I tested my changes on the spp2280 environment, using manually input data.
Please note due to my irregular hours I have destroyed this environment but it can be rebuilt using the existing concourse pipeline.
No BDD has been changed for this, though it may be necessary if current tests expect more than 2 decimal places of precision.
